site/static/main.js

```diff
@@ -1,36 +1,34 @@
 /* tslint:disable:no-unused-variable */
```

Comment:
Let's do this in a separate pr. If we want to compile a bundle, ts will be useful.